tests/integration_tests/tests/utils.py

```diff
@@ -87,8 +90,12 @@ def publish_event(queue, routing_key, event):
 
 def restore_provider_context():
     delete_provider_context()
-    client = create_rest_client()
-    client.manager.create_context(PROVIDER_NAME, PROVIDER_CONTEXT)
+    setup_app()
```

Comment:
Move `get_remote_storage_manager` from test_cases to utils into a separate function